src/main/resources/templates/templateDslBuilder.ftlh

```diff
@@ -151,6 +151,11 @@
 <#macro writeBuilderComplexField complexField>
   <#if complexField.complexType.name() == "COLLECTION">
       <#if complexField.fieldValidations??>
+          <#if complexField.fieldValidations.randomSize??>
+                <#assign randomSize = "${complexField.fieldValidations.randomSize}"/>
+          <#else >
+                <#assign collectionSize = ""/>
```

Comment:
What is the use for collectionSize variable?

Reply:
To be a forgotten replacement...